lib/express/server.js

```diff
@@ -70,24 +72,44 @@ async function server (configureRoutes, port, hostname = null, allowCors = true)
   });
 }
 
-function configureServer (app, configureRoutes, allowCors = true) {
+function normalizeBasePath (basePath) {
```

Comment:
Is it better to add tests to clarify the cases?

Reply:
yes, good idea to add some unit tests for this method